lib/table.js

```diff
@@ -3,6 +3,8 @@
 var wrap = require('word-wrap');
 
 var Table = require('cli-table2'),
+    argv  = require('yargs').argv,
+    _ = require('lodash'),
```

Comment:
'lodash' is not a dependency in package.json. Just depend on 'lodash.sortBy' (making sortProperty an argument you can skip _.isUndefined).